controllers/menuDiscountController.js

```diff
@@ -39,23 +44,54 @@ const getAllMenuDiscount = async (req, res) => {
             // })
             .populate({
                 path: "discountId",
-                select: "label percentDiscount"
+                select: "label percentDiscount",
             })
             .lean();
 
         if (!menuDiscounts?.length) {
             return res.status(400).json({ message: "No menu discounts found" });
         }
 
```

Comment:
@laubana same case here, an empty collection is not an error. It's just that it is empty. So, return 200, and then the empty menuDiscounts. Remove lines 52 to 54.

Reply:
I didn't change this.